plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestingPhoenixServer.java

```diff
@@ -89,7 +89,7 @@ private TestingPhoenixServer()
         this.conf.set("hbase.security.logger", "ERROR");
         this.conf.setInt(MASTER_INFO_PORT, -1);
         this.conf.setInt(REGIONSERVER_INFO_PORT, -1);
-        this.conf.setInt(HBASE_CLIENT_RETRIES_NUMBER, 1);
+        this.conf.setInt(HBASE_CLIENT_RETRIES_NUMBER, 15);
```

Comment:
The default value is 15 according to https://hbase.apache.org/book.html (`hbase.client.retries.number`). Can we instead remove this config?

Reply:
I would specify explicitly to use the same number between 4 and 5. The default number differs in two versions.